bouncer.go

```diff
@@ -102,6 +117,17 @@ func New(ctx context.Context, next http.Handler, config *Config, name string) (h
 	serverChecker, _ := ip.NewChecker(config.ForwardedHeadersTrustedIPs)
 	clientChecker, _ := ip.NewChecker(config.ClientTrustedIPs)
 
+	tlsConfig, err := getTLSConfigCrowdsec(*config)
+	if err != nil {
+		// should this error exit ?
```

Comment:
you know better than me, I think yes, if the key is configured

Reply:
I'll add an exit statement then, better crash right aways with a clear error message that not work as intended